projects/blenderbot2/agents/sub_modules.py

```diff
@@ -181,7 +181,7 @@ def __init__(self, opt: Opt):
             )
             assert isinstance(base_agent, TorchAgent)
             self.agents = [base_agent]
-            bsz = max(opt.get('batchsize', 1), opt.get('eval_batchsize', 1))
+            bsz = max(opt.get('batchsize') or 1, opt.get('eval_batchsize') or 1)
```

Comment:
Oddly, I was getting this error when trying this agent. This change was a solution to avoid it. But maybe we should 

![Screen Shot 2021-12-01 at 11 24 42 AM](https://user-images.githubusercontent.com/11262163/144292435-659f60da-b414-4c9b-accd-bf53fac4ff46.png)

Reply:
hmm interesting, I thought anything return from opt.get(XXX, 1) would never be None

Reply:
it can be `None` if the arg is in `opt` but no value is given